File: 1009.c
#include <stdio.h>
 
int main() { 
    char nome;
    double salarioFixo, vendas, total;

    scanf("%s %lf %lf", &nome, &salarioFixo, &vendas);
    total = salarioFixo + (vendas * 0.15);
    printf("TOTAL = R$ %.2f\n", total);

    return 0;
}

File: Iniciante/1009.java
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
 
        Scanner scanf = new Scanner(System.in);      
        
        String nome = scanf.nextLine();
        double salarioFixo = scanf.nextDouble();
        double vendas = scanf.nextDouble();
        double total = salarioFixo + (vendas * 0.15);       

        System.out.printf("TOTAL = R$ %.2f\n", total);
 
    }
 
}
